pages/test_analytics/javascript_collectors.md

```diff
@@ -58,6 +58,7 @@ To configure Jest:
         "reporters": ["default", "buildkite-test-collector/jest/reporter"]
```

Comment:
[Your suggestion](https://github.com/buildkite/docs/pull/2857/files#r1654093536) sounds even better then!

How about this then?

```suggestion
        "reporters": ["default", "buildkite-test-collector/jest/reporter"],
        "testLocationInResults": true
```

Reply:
OK, so we don't document the rationale for `"testLocationInResults": true`?

Reply:
Sorry I missed seeing that in your previous comment @malclocke !

Yes I agree that that's important to mention it, and so I've committed this information in a note underneath this JSON code snippet. (I was trying to do that directly through this PR, but had trouble getting the GitHub editor to do this, so I had to commit and push up from a local change.)